AppController/lib/cron_helper.rb

```diff
@@ -285,35 +285,143 @@ def self.fix_months(months)
   #   standard cron format. It cannot be applied to a crontab because it only
   #   specifies the frequency of the job, and not the action to perform.
   def self.convert_messy_format(schedule)
-    cron = ""
     splitted = schedule.split
 
-    unless splitted.length == 3 or splitted.length == 5
+    unless splitted.length == 3 || splitted.length == 5 || splitted.length == 7
```

Comment:
Could you put a comment on what we are getting here? Perhaps with a link to GAE cron?